app.js

```diff
@@ -62,64 +62,55 @@ if(process.env.VCAP_SERVICES){
 }
 logger.info("db type=="+dbtype);
 
-var routes = new require('./routes/index.js')(dbtype, authService,settings);
+var routes = new require('./routes/index.js')(dbtype, authService, settings);
```

Comment:
I would not define the routes this way in a proper porting. This is not a good example on how to structure _any_ Node.js applications (unfortunately). Check out https://github.com/fastify/fastify-example-twitter as structure example.
You can log this as a separate issue, as it's a refactoring that does not impact performance at all, but rather it's a more proper application structure.